scripts/ci/upload-autodoc.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
```

Comment:
I don't think we can produce autodocs without the contract being in Lorentz/Indigo.

This file is probably not useful to us either and should be removed.

Reply:
Removed this.